aspnetcore/host-and-deploy/docker/building-net-docker-images.md

```diff
@@ -0,0 +1,211 @@
+
+# Docker images for ASP.NET Core
+
+This tutorial shows how to run an ASP.NET Core app in Docker containers.
+
+In this tutorial, you:
+> [!div class="checklist"]
+> * Learn about Microsoft .NET Core Docker images 
+> * Download an ASP.NET Core sample app
+> * Run the sample app locally
+> * Run the sample app in Linux containers
+> * Run the sample app in Windows containers
+> * Build and deploy manually
+
+## ASP.NET Core Docker images
+
+For this tutorial, you download an ASP.NET Core sample application and run it in Docker containers. The sample works with both Linux and Windows containers.
+
+The sample Dockerfile uses the [Docker multi-stage build feature](https://docs.docker.com/engine/userguide/eng-image/multistage-build/) to build and run in different containers. The build and run containers are created from images that are provided in Docker Hub by Microsoft:
+
+* `dotnet/core/sdk`
+
+  The sample uses this image for building the app. The image contains the .NET Core SDK, which includes the Command Line Tools (CLI). The image is optimized for local development, debugging, and unit testing. The tools installed for development and compilation make this a relatively large image. 
+
+* `dotnet/core/aspnet` 
+
+   The sample uses this image for running the app. The image contains the ASP.NET Core runtime and libraries and is optimized for running apps in production. Designed for speed of deployment and app startup, the image is relatively small, so network performance from Docker Registry to Docker host is optimized. Only the binaries and content needed to run an app are copied to the container. The contents are ready to run, enabling the fastest time from `Docker run` to app startup. Dynamic code compilation isn't needed in the Docker model.
+
+## Prerequisites
+
+* [.NET Core 2.2 SDK](https://www.microsoft.com/net/core)
+
+* Docker client 18.03 or later
+
+  * Linux distributions
+     * [CentOS](https://docs.docker.com/install/linux/docker-ce/centos/)
+     * [Debian](https://docs.docker.com/install/linux/docker-ce/debian/)
+     * [Fedora](https://docs.docker.com/install/linux/docker-ce/fedora/)
+     * [Ubuntu](https://docs.docker.com/install/linux/docker-ce/ubuntu/)
+  * [macOS](https://docs.docker.com/docker-for-mac/install/)
+  * [Windows](https://docs.docker.com/docker-for-windows/install/)
+
+* [git](https://git-scm.com/download)
+
+## Download the sample application
+
+* Download the sample by cloning the [.NET Core Docker repository](https://github.com/dotnet/dotnet-docker): 
+
+  ```console
+  git clone https://github.com/dotnet/dotnet-docker
+  ```
+
+## Run the app locally
+
+* Navigate to the project folder at *dotnet-docker/samples/aspnetapp/aspnetapp*.
+
+* Run the following command to build and run the app locally:
+
+  ```console
+  dotnet run
+  ```
+
+* After the application starts, visit `http://localhost:5000` in a browser to verify that the app is working.
+
+* Press Ctrl+C at the command prompt to stop the app.
```

Comment:
at the command prompt or in the command prompt?

Reply:
at the command prompt or in the command prompt window.